### DISCUSSION TITLE
Jwt token undefined when following docs

### DISCUSSION THREAD

**Discussion by ChrisKneller** [fastapi/fastapi#7613] [Questions] [answered]
### Describe the bug

When following the guide on the docs here: https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/

When I try to execute the /users/me endpoint the token is undefined and so authentication fails and the page cannot be accessed.

### To Reproduce

Just follow the structure shown here https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/

### Environment

- OS: Windows
- FastAPI Version 0.54.1

### Additional context

Somebody else has experienced the problem here #1380

**Comment by ChrisKneller** [comment]
https://github.com/jpadilla/pyjwt/issues/405 looks to be potentially related

**Comment by ChrisKneller** [comment, marked as answer]
User error. Not a bug.

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue :+1:
